care/facility/api/serializers/patient_consultation.py

```diff
@@ -165,8 +184,46 @@ def update(self, instance, validated_data):
 
         consultation = super().update(instance, validated_data)
 
+        try:
+            health_details_data = self.context["request"].data[
+                "new_health_details"
+            ]
+            vaccination_history = health_details_data.pop(
+                "vaccination_history", []
+            )
+            PatientHealthDetails.objects.filter(
+                id=consultation.last_health_details.id
+            ).update(**self.context["request"].data["new_health_details"])
+            consultation.last_health_details = (
+                PatientHealthDetails.objects.get(
+                    id=consultation.last_health_details.id
+                )
+            )
+            vaccines = []
+            for vaccine in vaccination_history:
+                vaccines.append(
+                    Vaccine(
+                        health_details=consultation.last_health_details,
+                        **vaccine,
+                    )
+                )
+            if vaccines:
+                Vaccine.objects.bulk_create(vaccines, ignore_conflicts=True)
+            consultation.save(update_fields=["last_health_details"])
+            NotificationGenerator(
+                event=Notification.Event.PATIENT_HEALTH_DETAILS_UPDATED,
```

Comment:
I think we can ignore this notification, @gigincg the consultation notification should be enough right?

Reply:
I have talked with Gigin. He told me to remove this notification